eng/Directory.Build.Data.props

```diff
@@ -64,4 +66,8 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <NeutralLanguage>en-US</NeutralLanguage>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
All the other PackageReferences are coming in the targets file and I generally prefer them there instead of in the props.